Gordon360/Controllers/MembershipsController.cs

```diff
@@ -15,31 +15,14 @@ namespace Gordon360.Controllers
     public class MembershipsController : GordonControllerBase
     {
         private readonly IMembershipService _membershipService;
+        private readonly IAccountService _accountService;
         private readonly IActivityService _activityService;
 
-        public MembershipsController(CCTContext context, IActivityService activityService)
+        public MembershipsController(CCTContext context, IActivityService activityService, IAccountService accountService, IMembershipService membershipService)
```

Comment:
```suggestion
        public MembershipsController(IActivityService activityService, IAccountService accountService, IMembershipService membershipService)
```
Unnecessary since we are using DI for the rest of the services